src/hooks/usePersonalData/localStorageClient/localStorageClient.tsx

```diff
@@ -107,14 +108,28 @@ const setRecentSearch = async (id: string, type: 'video' | 'channel') => {
 
   writeToLocalStorage('recentSearches', newRecentSearches)
 }
+
+const dismissedMessages = promisify(() => readFromLocalStorage<DismissedMessage[]>('dismissedMessages') ?? [])
+const setDismissedMessages = async (id: string, add = true) => {
```

Comment:
If we want to use this pattern of one function, we will need to adjust the naming. it says `setDimissiedMessages` so  suggests you use it to set and array, however it accepts an id and boolean as args and modifies state of only one message. so `setDimissiedMessage` I guess works better